Flat/src/GeneralTree.cc

```diff
@@ -567,6 +573,26 @@ void GeneralTree::WriteTree(TTree *t) {
       Book("nJet_jesUp",&nJet_jesUp,"nJet_jesUp/I");
       Book("nJet_jesDown",&nJet_jesDown,"nJet_jesDown/I");
     }
+    Book("jot1PhiUp",&jot1PhiUp,"jot1PhiUp/F");
+    Book("jot1PhiDown",&jot1PhiDown,"jot1PhiDown/F");
+    Book("jot2PhiUp",&jot2PhiUp,"jot2PhiUp/F");
+    Book("jot2PhiDown",&jot2PhiDown,"jot2PhiDown/F");
+    if (!vbf) { // to avoid double booking
+      Book("jot1Phi",&jot1Phi,"jot1Phi/F");
+      Book("jot1Eta",&jot1Eta,"jot1Eta/F");
+      Book("jot1Pt",&jot1Pt,"jot1Pt/F");
+      Book("jot2Phi",&jot2Phi,"jot2Phi/F");
+      Book("jot2Eta",&jot2Eta,"jot2Eta/F");
+      Book("jot2Pt",&jot2Pt,"jot2Pt/F");
+      Book("jot1EtaUp",&jot1EtaUp,"jot1EtaUp/F");
+      Book("jot1EtaDown",&jot1EtaDown,"jot1EtaDown/F");
```

Comment:
This is not good. You're booking these branches for every analysis

Reply:
Also I don't get why things like `jot1Phi` are inside `if (vbf) { ... }` and `if (!vbf) { ... }`. That is needlessly complicated

Reply:
This is the only thing I haven't changed, sorry if it wasn't clear, but I didn't know how to do it without interfering with something else. Those variables are within the If(leptonic) statement, so that they won't be stored by any other analysis. On the other hand, if leptonic and vbf are set to true, I need to avoid double booking, that's why I need to do if(!vbf). Or is the double booking not a problem?